mote/.stylelintrc

```diff
@@ -0,0 +1,359 @@
+plugins:
+    - stylelint-selector-bem-pattern
+    - stylelint-scss
+rules:
+    plugin/selector-bem-pattern:
+        componentName: '[A-Z]+'
+        componentSelectors:
+            initial: '^\\.{componentName}(?:-[a-z]+)?$'
+        utilitySelectors: '^\\.u-[a-z]+$'
+
+    # Color (http://stylelint.io/user-guide/rules/#color)
+    color-hex-case: lower
+    color-hex-length: long
+    color-named: never
+    color-no-invalid-hex: true
+
+    # Font family (http://stylelint.io/user-guide/rules/#font-family)
+    font-family-name-quotes: always-where-required
+
+    # Font weight (http://stylelint.io/user-guide/rules/#font-weight)
+    font-weight-notation: numeric
+
+    # Function (http://stylelint.io/user-guide/rules/#function)
+    function-calc-no-unspaced-operator: true
+    function-comma-newline-after: never-multi-line
+    function-comma-newline-before: never-multi-line
+    function-linear-gradient-no-nonstandard-direction: true
+    function-max-empty-lines: 0
+    function-name-case: lower
+    function-parentheses-newline-inside: never-multi-line
+    function-parentheses-space-inside: never
+    function-url-quotes: always
+    function-whitespace-after: always
+
+    # Number (http://stylelint.io/user-guide/rules/#number)
+    number-leading-zero: never
+    number-max-precision: 3
+    number-no-trailing-zeros: true
+
+    # String (http://stylelint.io/user-guide/rules/#string)
+    string-no-newline: true
+    string-quotes: double
+
+    # Length (http://stylelint.io/user-guide/rules/#length)
+    length-zero-no-unit: true
+
+    # Time (http://stylelint.io/user-guide/rules/#time)
+    time-no-imperceptible: true
+
+    # Unit (http://stylelint.io/user-guide/rules/#unit)
+    unit-case: lower
+    unit-no-unknown: true
+
+    # Value (http://stylelint.io/user-guide/rules/#value)
+    value-keyword-case: lower
+    value-no-vendor-prefix: true
+
+    # Value list (http://stylelint.io/user-guide/rules/#value-list)
+    value-list-comma-newline-after: always-multi-line
+    value-list-comma-newline-before: never-multi-line
+    value-list-comma-space-after: always
+    value-list-comma-space-before: never
+    value-list-max-empty-lines: 0
+
+    # Custom property (http://stylelint.io/user-guide/rules/#custom-property)
+    # We don't currently use custom properties.
+
+    # Shorthand property (http://stylelint.io/user-guide/rules/#shorthand-property)
+    shorthand-property-no-redundant-values: true
+
+    # Property (http://stylelint.io/user-guide/rules/#property)
+    property-case: lower
+    property-no-unknown: true
+    property-no-vendor-prefix: true
```

Comment:
while we are on the point of prefix's have you managed to confirm that autoprefix is working?